### PR TITLE
Add Ukrainian translation

### PR DESCRIPTION
**Pull request by captain-yossarian**
Add Ukrainian translation

### CODE REVIEW

File: src/utility/translations.rs [hunk 1]
```diff
@@ -584,6 +638,7 @@ pub fn deep_sea_translation(language: Language) -> &'static str {
         Language::ES => "Para sumergirse en el tráfico de la red",
         Language::PL => "Aby zanurzyć się w ruchu sieciowym",
         Language::DE => "Um in den Netzwerkverkehr einzutauchen",
+        Language::UA => "Оригінальний, темний мотив Sniffnet",
```
Comment (GyulyVGC) [inline src/utility/translations.rs:641]: I was rapidly looking at the translations and found out that this one I equal to one of the preceding and I think it's not in the correct spot

**Review by GyulyVGC — CHANGES_REQUESTED** [review]

File: src/utility/translations.rs (hunk 1 above)
Comment (captain-yossarian) [reply src/utility/translations.rs:641]: Good catch, will fix

Comment (captain-yossarian) [reply src/utility/translations.rs]: @GyulyVGC  done

**Review by GyulyVGC — APPROVED** [review]

**Comment by GyulyVGC** [comment]
@all-contributors please add @captain-yossarian for translation.

**Pull request merged**